<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anime</class>
 <widget class="QWidget" name="Anime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>680</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S0</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S3</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S4</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S5</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="labelHint">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>761</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>现在是竞争期，由信号站N发送信号!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelHint2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>761</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>本次传输期需要发送帧的信号站(按顺序):</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>381</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S6</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S7</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_9" native="true">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>110</y>
         <width>61</width>
         <height>181</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>41</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>21</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>S8</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pbExit">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>490</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>exit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbLove">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>430</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>continue</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbAlways">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>370</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>auto</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_18">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_19">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>要</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>110</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
